### PR TITLE
workspace changes for imports redesign

### PR DESCRIPTION
**Pull request by JohanWiltink**
Please review carefully!

### CODE REVIEW

**Comment by kazk** [comment]
I actually already updated #64

**Comment by kazk** [comment]
I also released v1.0.0-rc.4 so we can update `workspace/` and test.

File: workspace/test.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
-import { assert, config as chaiConfig } from "chai";
-chaiConfig.truncateThreshold = 0;
+// test.js
 
-import * as LC from "@codewars/lambda-calculus";
-import { solution } from "./files.js"; // /workspace/files.js
+import * as LC from "./lc-test.js";
+LC.config.truncateThreshold = 0;
+LC.configure({ purity: "Let", numEncoding: "Church" });
```
Comment (kazk) [inline workspace/test.js:5]: I don't think this works. `lc-test.js` has `export * as LC from "@codewars/lambda-calculus";` so this needs to be `LC.LC.configure`.

See https://github.com/codewars/lambda-calculus/pull/64/files#diff-048f6c0b471ab2151a053dc74ea5e88ae31b8c871aee8b4f8582bda6a6e9cd78

Comment (JohanWiltink) [reply workspace/test.js:5]: Having stuff in `LC.LC.` is not my intention

Comment (kazk) [reply workspace/test.js:5]: Can you suggest changes to my PR? That works and the main difference is `chai.config`.

Comment (JohanWiltink) [reply workspace/test.js:5]: I'm awfully sorry but I can't. I can suggest what files should look like in the end, but I don't know how to suggest a diff on a diff.

Comment (kazk) [reply workspace/test.js:5]: You can just tell me what's wrong with that PR.

Comment (JohanWiltink) [reply workspace/test.js:5]: Your PR changes things that are then changed again by my PR, but from a different starting point.

I fear that will break things.

Comment (kazk) [reply workspace/test.js:5]: ? We're not merging both. I'll close this to avoid confusion.

Comment (JohanWiltink) [reply workspace/test.js:5]: Then we merge yours I guess, and then I think some more changes are necessary.

Comment (kazk) [reply workspace/test.js:5]: Like what? It should contain the equivalent code now. I changed to export `chai.config` and doesn't contain the import bug in this one.

#64 should have everything to create a new image.

**Pull request closed without merging**